### ISSUE TITLE
What should constraining the block-progression direction do?

### ISSUE THREAD

**Issue by travisleithead** [WICG/canvas-formatted-text#43]
The explainer's data model allows height and width to be specified. Depending on the writing-mode specified on the context object, either of these dimensions could become the block-progression direction. We know that we want the inline direction to wrap content. But what should happen when the limit specified in the block direction is reached?

* Is there a "fit" model that choses when to overflow the box, already specified (or in draft) in CSS fragmentation? (Partial lines would be "pushed" to a following container/dropped alltogether?)
* Should content be clipped exactly at that pixel boundary (relevant when rendering)
* Should content just continue in the block-progression dimension inrrespective of the limit specified (as if it renders offscreen)--effectively ignoring the limit?

This was brought up in review of [PR 39](https://github.com/WICG/canvas-formatted-text/pull/39#discussion_r798150213)

**Comment by dlibby-** [comment]
Intuitively, I don't think clipping makes much sense (and default for CSS is overflow: visible anyhow). If we format the runs and expose line metrics, we'd have to lay everything out anyways. So maybe we apply that principle, and leave clipping up to the canvas renderer if necessary?

**Comment by travisleithead** [comment]
Yes, that makes sense to me. It could also be interesting to expose a boolean line metric flag that would be true when the particular line intersects with or is beyond the provided bounding box of the provided constraints.

**Comment by fantasai** [comment]
You might want to have it lay out as many lines as will fit, and then stop, and allow the caller to continue formatting the paragraph in another function call (or not format it, similar to using line-clamp).

**Comment by fantasai** [comment]
But also, if you're not sure what you want to do, maybe you don't offer this parameter. :)
